ci/terraform/modules/public-api-gateway/variables.tf

```diff
@@ -51,3 +51,8 @@ variable "waf_arns" {
   description = "The ARNs of any WAFs to attach to the API Gateway"
   type        = list(string)
 }
+
+variable "domain_name" {
```

Comment:
Is this use in [this file](https://github.com/govuk-one-login/authentication-api/blob/03f1103df1f7985ef6363a07b7b7d529081f6187/ci/terraform/modules/public-api-gateway/domain-mapping.tf) the reason to add this. Should we expect an error or warning without this variable declared?

Reply:
Yeah, that is correct.

I don't believe this module is actually in use anywhere, it's there mostly because we have a private one, so it makes sense to also have the `public` version of the module implemented.

When creating the module, apparently I forgot to add this variable to `variables.tf` and so when modifying the `extra_tags` variable, I had a red squiggly, so fixed this variable while here, to avoid any validation errors.

Happy to drop the commit that adds this variable if it's creating noise in the context of this PR

Reply:
Thanks, makes sense. No need to drop from this PR 👍